uniprot-config/src/main/resources/search-fields-config/uniparc-search-fields.json

```diff
@@ -132,9 +132,9 @@
     "label": "Active",
     "itemType": "SINGLE",
     "fieldType": "GENERAL",
-    "dataType": "STRING",
+    "dataType": "ENUM",
     "description": "Search by active name",
-    "example": "sample active",
+    "example": "Yes",
```

Comment:
This is not boolean (I also got confused with it as well), it is active databases, so, the correct example would be "Gene3D"